jobs/config.json

```diff
@@ -2590,14 +2590,15 @@
       "--extract=ci/latest",
       "--extract=ci/latest-1.7",
       "--gcp-zone=us-central1-f",
+      "--ginkgo-parallel",
       "--provider=gce",
-      "--test_args=--kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8",
-      "--timeout=900m",
+      "--test_args=--ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8",
+      "--timeout=120m",
       "--upgrade_args=--ginkgo.focus=\\[Feature:ClusterUpgrade\\] --upgrade-target=ci/latest"
     ],
     "scenario": "kubernetes_e2e",
     "sigOwners": [
-      "UNKNOWN"
+      "sig-cluster-lifecycle"
```

Comment:
Whoa. This is probably not right. We only own a very small subset of the tests that this runs.

cc @kubernetes/sig-cluster-lifecycle-pr-reviews @roberthbailey

Reply:
eh maybe sig-release I guess?